proposals/physical-lighting/README.md

```diff
@@ -32,14 +32,19 @@ class PhysicalLightIlluminantAPI (
 )
 {
     token physical:illuminant = "white" (
-        doc = """The illuminant spectrum of the light in spectral radiance. The default of "white" specifies that the spectrum matches the whitepoint of the rendering color space. "blackbody" specifies a blackbody spectrum of temperature defined by "colorTemperature", and "custom" specifies that the spectrum in "physical:customIlluminant" should be used.
-        "white" and "blackbody" are defined at 5nm intervals over the domain [380nm, 780nm]."""
+        doc = """The illuminant spectrum of the light in spectral radiance. The default of "white" specifies that the spectrum
+                 matches the whitepoint of the rendering color space. "blackbody" specifies a blackbody spectrum of temperature
+                 defined by "colorTemperature", and "custom" specifies that the spectrum in "physical:customIlluminant" should
+                 be used. "white" and "blackbody" are defined at 5nm intervals over the domain [380nm, 780nm]."""
         allowedTokens = ["white", "blackbody", "custom"]
```

Comment:
is it worth adding a daylight illuminant as well, since that is skewed from blackbody?

Reply:
I did consider this... it ends up being a question of "where do you stop"? Though I guess there's a strong argument for at least D65 and D50